hrs-portlets-webapp/src/main/java/edu/wisc/portlet/hrs/web/HrsControllerBase.java

```diff
@@ -64,6 +73,24 @@ public final String getNavLinks(PortletRequest request) {
     }
     
     /**
+     * The generic Message if you want to override the default in 
+     * messages.properties
+     */
+    @ModelAttribute("genericErrorMessage")
+    public final String getGenericErrorMessage(PortletRequest request){
+        final PortletPreferences preferences = request.getPreferences();
+        ClassLoader loader = Thread.currentThread().getContextClassLoader();
+        Properties props = new Properties();
+        InputStream resourceStream = loader.getResourceAsStream(MESSAGE_PROPERTIES_FILE_NAME);
```

Comment:
Instead of implementing this reading and parsing of `messages.properties` here, I suspect there's some way to get a `MessageSource` from the `ApplicationContext` or have this message value injected or so.